### PR TITLE
feat: ROCM 5.0 

### PR DESCRIPTION
**Pull request by azhou-determined**
## Description
Support for ROCM 5.0

## Checklist

- [ ] Bump VERSION to make the pushed images are tagged with the right version.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,7 +44,9 @@ export CPU_TF26_ENVIRONMENT_NAME := $(CPU_PREFIX)tf-2.6$(CPU_SUFFIX)
 export GPU_TF26_ENVIRONMENT_NAME := $(CUDA_112_PREFIX)tf-2.6$(GPU_SUFFIX)
 export CPU_TF27_ENVIRONMENT_NAME := $(CPU_PREFIX)tf-2.7$(CPU_SUFFIX)
 export GPU_TF27_ENVIRONMENT_NAME := $(CUDA_112_PREFIX)tf-2.7$(GPU_SUFFIX)
+
 export ROCM_TORCH_TF_ENVIRONMENT_NAME := $(ROCM_42_PREFIX)pytorch-1.9-tf-2.5-rocm
```
Comment (ioga) [inline Makefile:48]: is the old one still useful? maybe remove it?

**Review by ioga — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (azhou-determined) [reply Makefile:48]: i think we should just leave it for now, since i'm not sure how many users on actually on the latest ROCM

**Review by ioga — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (ioga) [reply Makefile:48]: I suspect about zero

Comment (azhou-determined) [reply Makefile]: good enough for me. removed.

**Review by ioga — APPROVED** [review]

**Review by ioga — APPROVED** [review]

**Pull request merged**